Python/omevv/v1/omevv_apis_client/models/job_execution_history.py

```diff
@@ -89,8 +89,11 @@ def from_dict(cls: Type[T], src_dict: Dict[str, Any]) -> T:
         end_time: Union[Unset, datetime.datetime]
         if isinstance(_end_time, Unset):
             end_time = UNSET
-        else:
-            end_time = isoparse(_end_time)
+        else:  # Added below code to handle when next_is is none
```

Comment:
typo- comment to include proper variable name

Reply:
@pavithra-mahadev : Corrected it.